packages/rxjs/tests/RxJS.test.ts

```diff
@@ -210,3 +210,51 @@ describe("RxJS", () => {
     assert.deepEqual(sub.closed, true);
   });
 });
+
+describe("effectToObservable", () => {
+  interface Counters<A> {
+    values: A[];
+    errors: unknown[];
+  }
+  const makeCounters = <A>(): Counters<A> => ({
+    errors: [],
+    values: []
+  });
+
+  const test = <A>(eff: T.Effect<unknown, never, A>) =>
+    Do(T.effect)
+      .bind("counters", T.pure(makeCounters<A>()))
+      .bind("obs", T.pure(O.effectToObservable(eff)))
+      .bindL("res", ({ obs, counters }) =>
+        T.asyncTotal(cb => {
+          obs.subscribe(
+            n => {
+              counters.values.push(n);
+            },
+            e => {
+              counters.errors.push(e);
+              // cb("ok");
+            },
+            () => {
+              cb("ok");
+            }
+          );
+          return () => {};
+        })
+      )
+      .return(r => r.counters);
+
+  it("effectToObservable returns value", async () => {
+    const counters = await T.runToPromise(test(T.pure("a")));
+    assert.deepEqual(counters.values, ["a"]);
+    assert.deepEqual(counters.errors, []);
+  });
+
+  it("effectToObservable returns error", async () => {
+    const counters = await T.runToPromise(
+      test(T.raiseError("err") as any) // forces accepting errors with any
```

Comment:
?

Reply:
Signature prevent to make Observable from failing Effect so that cast is to force Errors that are not allowed by the type checker (like execution errors)

Reply:
@mikearnaldi  May I write a throw?

Reply:
everytime you throw you end up in hell...

Reply:
an execution error will never be catched properly to be propagated here, it will resolve to an abort in the fiber

Reply:
ok so you're saying no T.raiseError would ever be generated so no reason for this test as it test an impossible case, right?

Reply:
make sense, will remove it